exe/pr

```diff
@@ -84,4 +85,5 @@ sh "git push -u origin #{branch}"
 
 puts "Creating pull request"
 ENV["PR_BODY"] = body_lines.join("\n")
-sh "gh pr create --draft --title '#{title}' --body \"$PR_BODY\" --base #{base_branch}"
+
+sh "gh pr create --draft --title #{Shellwords.escape(title)} --body #{Shellwords.escape(body)} --base #{base_branch}"
```

Comment:
if this actually works for multiline strings (and during my experiments it did not), then the line above is not needed anymore

Reply:
@JensRavens @jmarsh24 : Was that resolved?

Reply:
that's exactly the line you requested an RFC, so it's still open